src/runtime-rs/crates/hypervisor/src/dragonball/inner_device.rs

```diff
@@ -84,6 +84,9 @@ impl DragonballInner {
         info!(sl!(), "remove device {} ", device);
 
         match device {
+            DeviceType::Network(_network) => {
+                todo!()
```

Comment:
If dbs support this we should implement this, since this is called on `detach()`. Otherwise it would panic here.

Reply:
fixed